data/data/openstack/variables-openstack.tf

```diff
@@ -252,3 +240,11 @@ Block of IP addresses used by the VPC.
 This should not overlap with any other networks, such as a private datacenter connected via Direct Connect.
 EOF
 }
+
+variable "openstack_trunk_support" {
+  type = "string"
```

Comment:
nit: Any specific reason why you went with `string` instead of bool? I'd rather use `bool` if there's no specific reason to use `string`.

Reply:
According to the terraform [documentation](https://www.terraform.io/docs/configuration/variables.html#type) the valid types are "string, list, and map". There is no example of `bool` in the repository and digging around I saw that on 0.11 strings are the type supposed to be used for boolean values (when containing "true", "1", "false", "0"). Thankfully, this will change with [0.12](https://github.com/hashicorp/terraform/issues/6254).